app/code/RLTSquare/Ccq/Console/Command/HelloWorld.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RLTSquare\Ccq\Console\Command;
+
+use Exception;
+use Magento\Framework\MessageQueue\PublisherInterface;
+use Magento\Framework\Serialize\Serializer\Json;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * class for helloworld command
+ */
+class HelloWorld extends Command
+{
+    /**
+     * @const
+     */
+    private const VAR1 = 'var1';
+    /**
+     * @const
+     */
+    private const VAR2 = 'var2';
+    /**
+     * @const
+     */
+    private const TOPIC = 'rltsquare_hello_world';
+    /**
+     * @var PublisherInterface
+     */
+    private PublisherInterface $publisher;
+    /**
+     * @var LoggerInterface
+     */
+    private LoggerInterface $logger;
+    /**
+     * @var Json
+     */
+    private Json $json;
+
+    /**
+     * @param LoggerInterface $logger
+     * @param PublisherInterface $publisher
+     * @param string|null $name
+     */
+    public function __construct(
+        LoggerInterface $logger,
+        PublisherInterface $publisher,
+        Json $json,
+        string $name = null
+    ) {
+        $this->logger = $logger;
+        $this->publisher = $publisher;
+        $this->json = $json;
+        parent::__construct($name);
+    }
+
+    /**
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this->setName('rltsquare:hello:world');
+        $this->setDescription('RLTSquare Training Unit2 Command');
+        $this->setDefinition([
+            new InputOption(
+                self::VAR1,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'var1'
+            ),
+            new InputOption(
+                self::VAR2,
+                null,
+                InputOption::VALUE_REQUIRED,
+                'var2'
+            )
+        ]);
+        parent::configure();
+    }
+
+    /**
+     * method to display command output & publish a message to a specific queue
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     *
+     * @return int
+     */
+    protected function execute(
+        InputInterface $input,
+        OutputInterface $output
+    ): int {
+        $exitCode = 0;
+        try {
+            if (($var1 = $input->getOption(self::VAR1)) and ($var2 = $input->getOption(self::VAR2))) {
```

Comment:
Code refactoring - sometime you use "and", sometime "&&"; opt for one and be consistent ... ideally "&&".